gallery/src/components/NxCollapsibleItems/NxCollapsibleItemsDisabledExample.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { NxCollapsibleItems, NxCollapsibleItemsChild } from '@sonatype/react-shared-components';
+
+function NxCollapsibleItemsDisabledExample() {
+
+  return (
+    <NxCollapsibleItems isOpen={true} disabled={true} triggerContent="All Items">
+      <NxCollapsibleItemsChild>Test1</NxCollapsibleItemsChild>
```

Comment:
Maybe a good time to bring this more in line with other subcomponents and change it to `NxCollapsibleItems.Child` or something

Reply:
> Maybe a good time to bring this more in line with other subcomponents and change it to `NxCollapsibleItems.Child` or something

Added here :)
d163276c542c5faaa1ef4f20bdd8b10c0ce101ae